public/components/datasources/components/manage/manage_data_connections_table.tsx

```diff
@@ -38,11 +38,13 @@ import { DataConnectionsDescription } from './manage_data_connections_descriptio
 import { getRenderCreateAccelerationFlyout } from '../../../../../public/plugin';
 import { InstallIntegrationFlyout } from './integrations/installed_integrations_table';
 import { redirectToExplorerS3 } from './associated_objects/utils/associated_objects_tab_utils';
+import { checkIsConnectionWithLakeFormation } from '../../utils/helpers';
 
 interface DataConnection {
   connectionType: DatasourceType;
   name: string;
   dsStatus: DatasourceStatus;
+  isConnectionWithLakeFormation: boolean;
```

Comment:
Can we check the same logic with `connectionType: DatasourceType;`? Like `PROMETHEUS` or `S3GLUE`?

Reply:
Not today, we don't have a separate type for S3 connections with support for lake formation